client/package.json

```diff
@@ -67,7 +68,7 @@
     "workbox-webpack-plugin": "4.3.1"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "PORT=3001 node scripts/start.js",
```

Comment:
Doesn't need to be in the PR, but can we put the PORT into start.js instead of passing it in? That way, we should be able to set the port based on environment variables (via .env file), instead of having to pass it in.